### DISCUSSION TITLE
ImportError

### DISCUSSION THREAD

**Discussion by JunqiaoZhu** [gelles-brandeis/tapqir#397] [Q&A] [answered]
Hi there!

I encountered a problem with "ImportError" when running the program to fit the tutorial data. The "ImportError" showed up after the program reached the convergent iteration and completed fitting. It looks like this error prevented outputting the cosmos-summary.csv and cosmos-params.tpqr files, and I couldn't find those expected files in google drive for next step analysis. Could you please let me know the reason of the error? I have attached the data file as pdf format for you to review since the ipynb format is not supported to upload. Thank you!

[20221117_tutorial_error.pdf](https://github.com/gelles-brandeis/tapqir/files/10036315/20221117_tutorial_error.pdf)

**Comment by ordabayevy** [comment, marked as answer]
Hi @JunqiaoZhu . Thanks for trying out Tapqir!

Sorry for the late reply. The error looks similar to when the Google colab runtime is not restarted after the installation (`Runtime -> Restart runtime`). If you haven't please could you try that out and let me know if the issue is fixed?

**Reply by JunqiaoZhu** [reply]
Hello Yerdos, it worked out for me. Thanks very much for your help!
